pages/guideforms.py

```diff
@@ -642,6 +642,33 @@ def validate(self, value):
             'https://github.com/WICG/idle-detection/blob/main/HOWTO.md'
             '">Example 2</a>.')),
 
+    'dt_milestone_desktop_start': forms.IntegerField(
+        required=False, label='DevTrail on desktop',
+        widget=forms.NumberInput(attrs={'placeholder': 'Milestone #'}),
+        help_text=('First milestone that will allow developers to try out '
```

Comment:
```suggestion
        help_text=('First milestone that allows developers to try '
```

Reply:
Follows the best practice of omitting needless words, making the sentence small-screen friendly.

Reply:
Done.